src/inspector_agent.cc

```diff
@@ -567,12 +578,30 @@ void AgentImpl::WaitForDisconnect() {
                            v8::ReadOnly).FromJust();                          \
   } while (0)
 
+void AddCommandLineAPIMethod(const v8::FunctionCallbackInfo<v8::Value>& info) {
```

Comment:
@jkrems Yeah, for top-level functions (i.e. in namespace scope, like this one), adding `static` basically means that the function will only be visible in the file in which it is defined. I think it’s generally good practice to have any top-level functions either appear in a header file to make clear that they are shared between files/part of the public C++ API, or be `static` to indicate that they are intended to be local. You can grep `src/` for `FunctionCallbackInfo<Value>& args`, most of these functions are static.

(Also, yeah, no doubt that assigning multiple distinct meanings to `static` is not an ideal thing for C++. :smile:)